x-pack/plugins/saved_objects_tagging/public/components/edition_modal/create_or_edit_modal.tsx

```diff
@@ -139,6 +141,12 @@ export const CreateOrEditModal: FC<CreateOrEditModalProps> = ({
                     onClick={() => setColor(getRandomColor())}
                     size="xs"
                     style={{ height: '18px', fontSize: '0.75rem' }}
+                    aria-label={i18n.translate(
+                      'xpack.savedObjectsTagging.management.createModal.color.randomizeAriaLabel',
+                      {
+                        defaultMessage: 'Randomize tag color',
+                      }
+                    )}
```

Comment:
We should provide an `aria-label` with more info here because the action `Randomize` means very little out of context.